editoast/src/views/infra/auto_fixes.rs

```diff
@@ -132,14 +156,20 @@ pub enum AutoFixesEditoastError {
     #[error("Failed trying to apply fixes")]
     #[editoast_error(status = 500)]
     FixTrialFailure { source: InternalError },
+    #[error("Failed to find the error's object")]
+    #[editoast_error(status = 500)]
+    MissingErrorObject { error: InfraError },
```

Comment:
minor (naming): it would feel clearer to me
```suggestion
    #[error("Failed to find the object subject of the error")]
    #[editoast_error(status = 500)]
    ErrorSubjectNotFound { error: InfraError },
```

Other ideas (not for now IMO):
* Maybe specifying type+id would help, too, not being misled by reference.
* Anyway, it feels like the main responsibility of this EditoastError should be in InfraCache as it should be a common pattern to look for objects there. But I'm currently doing a partial job on that, and I might propose something for this case too.

Reply:
I'll skip this if that's ok (I find it confusing to introduce object + subject).
I agree about the InfraCache, there should probably be an interface to get a given object.